source/_integrations/roon.markdown

```diff
@@ -49,3 +50,71 @@ Transfer playback from one player to another.
 | ---------------------- | -------- | ----------------------------- |
 | `entity_id`            | yes      | id of the source player.      |
 | `transfer_id`          | no       | id of the destination player. |
+
+## Roon endpoint volume control via Home Assistant
+
+For media players that are not fully integrated into roon, it is possible to use Home Assistant to implement a volume control service. This allows the native Roon apps to change the volume of an endpoint via automation in Home Assistant.
+
+For example if you have an amplifier where the volume can be controlled by Home Assistant (perhaps via an integration, or using an IR Blaster) you can have the roon apps use these to change volume.
+
+### Configuration in Roon
+
+The first step is to tell Roon to use Home Assistant to make volume changes. You do this in the roon app. First select the zone you wish to control; click on the cog item to enter Zone Setup, and then select Device Setup.
+
+![Roon volume options](/images/integrations/roon/roon_volume_options.png)
+
+One of the options shown is Volume Control. Alongside the normal roon options (such as DSP Volume) you will see a series of options called Home Assistant: ~Zone Name~. Choose the option that matches the zone you are configuring.
+
+The Roon volume control will now show plus and minus buttons rather that a volume control slider.
+
+### Automation in Home Assistant
+
+Clicking on the plus and minus buttons in Roon can now trigger an automation in the volume control entity matching the media_player in Home Assistant.
+
+In this autimation you can use Home Assistant services to provide `volume_up` and `volume_down`.
+
+Here is an example automation using an IR blaster to control `media_player_study`
+
+```yaml
+alias: Roon Study Volume
+description: ""
+trigger:
+  - platform: state
+    entity_id:
+      - event.study_roon_volume
+action:
+  - choose:
+      - conditions:
+          - condition: state
+            entity_id: event.study_roon_volume
+            state: volume_up
```

Comment:
The state of the event entity is the timestamp of the last event. We can check the state attribute event type instead.

Reply:
I think this code is already checking the state entity attribute (see the attribute key `event_type` below).

The HA condition documentations shows:

```yaml
condition:
  condition: state
  entity_id: climate.living_room_thermostat
  attribute: fan_mode
  state: "auto"
```

Which I think is the same.

Happy to be corrected if I've misunderstood.

Reply:
Yes, you're correct. I'd switch places of this line and the one below so we start by saying the entity and its attribute and end with what that should be.